ITrade/SendOfferToSteamId.md

```diff
@@ -15,7 +15,7 @@ API key required.
 
 Parameter | Type | Required   | Description
 --------- | -----| :--------: | -----------
-twofactor_code | int | + | 2FA Auth Code
+twofactor_code | string | + | 2FA Auth Code
```

Comment:
How come?

Reply:
Ok, i saw it in SendOffer and want to follow this standard but you change it here (https://github.com/OPSkins/trade-opskins-api/commit/570dc5ee7da90e0367874243e58ef103fe880cca) ;)